xdsl/dialects/memref.py

```diff
@@ -472,6 +472,7 @@ class Global(IRDLOperation):
     sym_visibility: StringAttr = prop_def(StringAttr)
     type: Attribute = prop_def(Attribute)
     initial_value: Attribute = prop_def(Attribute)
+    alignment: AnyIntegerAttr | None = opt_prop_def(AnyIntegerAttr)
```

Comment:
```suggestion
    alignment = opt_prop_def(Annotated[IntegerAttr[IntegerType], i64])
```
Does this work by any chance?

Reply:
yup!